packages/datagateway-common/src/detailsPanels/dls/visitDetailsPanel.component.test.tsx

```diff
@@ -132,137 +148,144 @@ describe('Visit details panel component', () => {
       },
     ];
 
-    const wrapper = createWrapper();
-    expect(wrapper.find('VisitDetailsPanel').props()).toMatchSnapshot();
+    const { asFragment } = renderComponent({ rowData });
+    expect(asFragment()).toMatchSnapshot();
   });
 
-  it('renders publications tab and text "No publications" when no data is present', () => {
+  it('should show publications tab and text "No publications" when no data is present', async () => {
     rowData.publications = [];
-    const wrapper = createWrapper();
+
+    renderComponent({ rowData });
+
     expect(
-      wrapper
-        .find('[data-testid="visit-details-panel-no-publications"]')
-        .exists()
-    ).toBeTruthy();
+      await screen.findByText(
+        'investigations.details.publications.no_publications'
+      )
+    ).toBeInTheDocument();
   });
 
-  it('renders samples tab and text "No samples" when no data is present', () => {
+  it('should show samples tab and text "No samples" when no data is present', async () => {
     rowData.samples = [];
-    const wrapper = createWrapper();
+    renderComponent({ rowData });
     expect(
-      wrapper.find('[data-testid="visit-details-panel-no-samples"]').exists()
+      await screen.findByText('investigations.details.samples.no_samples')
     ).toBeTruthy();
   });
 
-  it('renders users tab and text "No users" when no data is present', () => {
+  it('should show users tab and text "No users" when no data is present', async () => {
     rowData.investigationUsers = [];
-    const wrapper = createWrapper();
+    renderComponent({ rowData });
     expect(
-      wrapper.find('[data-testid="visit-details-panel-no-name"]').exists()
-    ).toBeTruthy();
-  });
-
-  it('calls useInvestigationDetails and useInvestigationSize hooks on load', () => {
-    createWrapper();
-    expect(useInvestigationDetails).toHaveBeenCalledWith(rowData.id);
-    expect(useInvestigationSize).toHaveBeenCalledWith(rowData.id);
```

Comment:
And just tagging onto my last comment, because of the changes we can no longer test these are being called (or we can but have chosen not to). So would be nice to have these back